File: AP1/Jutge/Python/trigonometry.py
from easyinput import read
from math import radians, sin, cos

x = read(float)

while x == x:
    a = sin(radians(x))
    b = cos(radians(x))
    print("{:.6f}".format(a),"{:.6f}".format(b))
    x = read(float)

File: AP1/Jutge/Seqüències/Trigonometry.cc
#include <cmath>
#include <iostream>
#include <math.h>
using namespace std;

void convertir_radiantes(double& grados)
{
    grados = grados * M_PI / 180;
}

int main()
{
    cout.setf(ios::fixed);
    cout.precision(6);

    double x;

    while (cin >> x) {
        convertir_radiantes(x);

        cout << sin(x) << " " << cos(x) << endl;
    }
}
